interfaces/dom.idl

```diff
@@ -108,50 +109,45 @@ Document includes ParentNode;
 DocumentFragment includes ParentNode;
 Element includes ParentNode;
 
-
 interface mixin NonDocumentTypeChildNode {
   readonly attribute Element? previousElementSibling;
   readonly attribute Element? nextElementSibling;
 };
 Element includes NonDocumentTypeChildNode;
 CharacterData includes NonDocumentTypeChildNode;
 
-
 interface mixin ChildNode {
-  [Unscopable] void before((Node or DOMString)... nodes);
-  [Unscopable] void after((Node or DOMString)... nodes);
-  [Unscopable] void replaceWith((Node or DOMString)... nodes);
-  [Unscopable] void remove();
+  [CEReactions, Unscopable] void before((Node or DOMString)... nodes);
+  [CEReactions, Unscopable] void after((Node or DOMString)... nodes);
+  [CEReactions, Unscopable] void replaceWith((Node or DOMString)... nodes);
+  [CEReactions, Unscopable] void remove();
 };
 DocumentType includes ChildNode;
 Element includes ChildNode;
 CharacterData includes ChildNode;
 
-
 interface mixin Slotable {
   readonly attribute HTMLSlotElement? assignedSlot;
 };
 Element includes Slotable;
 Text includes Slotable;
 
-
 [Exposed=Window]
 interface NodeList {
   getter Node? item(unsigned long index);
   readonly attribute unsigned long length;
-//  iterable<Node>;
+  iterable<Node>;
```

Comment:
This and the DOMTokenList change causes new failing tests due to this:
https://github.com/w3c/web-platform-tests/blob/9192f4b6c633f7f00ad0c1b983cbed33fff6be82/resources/idlharness.js#L2047

It fails for entries(), keys() and values() for each of NodeList and DOMTokenList, so 6 failing tests.

The per-spec behavior is defined by https://heycam.github.io/webidl/#es-iterable and since this is a "value iterator" these assertions could be made:
* `Array.prototype.entries===NodeList.prototype.entries`
* `Array.prototype.keys===NodeList.prototype.keys`
* `Array.prototype.values===NodeList.prototype.values`

That does in fact hold true in Chrome and Safari (maybe more) and explains why `NodeList.prototype.entries.call({})` should't throw TypeError but actually return something.

So, to merge this we need to at least make idlharness.js not assert that this should throw.

Reply:
Sounds reasonable - Can you write a PR that makes said change?

Reply:
I wrote https://github.com/w3c/web-platform-tests/pull/9790 back then, but then wasn't sure it worked at all. Needs tests I guess.

Reply:
As you described, [Chromium](https://test-results.appspot.com/data/layout_results/linux_chromium_rel_ng/150187/site_per_process_webkit_layout_tests%20%28with%20patch%29/layout-test-results/results.html) shows a bunch of regressions due to the issue. Are you able to rebase/get #9790 up to date?